packages/dropdown/src/useDropdown.ts

```diff
@@ -13,9 +16,9 @@ export const useDropdown = () => {
   }
 }
 
-export const useDropdownDomEvent = (dropdownChildren, triggerElm, _instance) => {
-  const menuItems = ref(null)
-  const menuItemsArray = ref(null)
+export const initDropdownDomEvent = (dropdownChildren, triggerElm, _instance) => {
+  const menuItems = ref<Nullable<NodeList>>(null)
```

Comment:
I saw you constantly casting `NodeList` to `HTMLButtonElement`, why not marking them as `HTMLButtonElement[]` ?